api/contrib/envoy/extensions/filters/http/language/v3alpha/language.proto

```diff
@@ -33,4 +33,7 @@ message Language {
     unique: true
     items {string {min_len: 2}}
   }];
+
+  // Clear the route cache for the current request to recalculate the route.
```

Comment:
Probably worth elaborating on this comment since it generates user-facing documentation, e.g.
```
If the x-language header is altered, clear the route cache for the current request.
This should be set if the route configuration may depend on the x-language header.
Otherwise it should be unset to avoid the performance cost of route recalculation.
```